Source/ACE.Server/WorldObjects/WorldObject_Use.cs

```diff
@@ -99,9 +99,18 @@ public virtual void OnActivate(WorldObject activator)
             // if ActivationTarget is another object,
             // should this be checking the ActivationResponse of the target object?
 
-            // default use action
-            if (ActivationResponse.HasFlag(ActivationResponse.Use))
+            // Chests with 'ActivationResponse - CastSpell' should still ActOnUse
+            if (WeenieType == WeenieType.Chest && Active == 65535)
```

Comment:
0xFFFF

Reply:
I was going to change it to a descriptive Enum